src/components/common/mixins/forms/types.ts

```diff
@@ -78,12 +81,12 @@ declare class BaseFormAssociatedElement {
    * the {@link BaseFormAssociatedElement.invalid | `invalid`} attribute on the control and the associated styles
    * if the element has completed the first update cycle or it has been interacted with by the user.
    */
-  protected _validate(message?: string): void;
+  protected _validate(message?: string, isUserSet?: boolean): void;
 
   /**
    * Executes the component's validators and updates the internal validity state.
    */
-  protected _updateValidity(message?: string): void;
+  protected _updateValidity(message?: string, isUserSet?: boolean): void;
```

Comment:
Are those some sort of leftover? Don't see any code calling in with the second param nor is it implemented since `BaseFormAssociated` doesn't actually have a connection to this type.